tests/itest_specs/view_functions/liquidity_pool_aggregate.cairo

```diff
@@ -40,16 +40,16 @@ namespace LPAggregateViewFunctions {
         assert poolinfo[0].pool.quote_token_address = myusd_addr;
         assert poolinfo[0].pool.base_token_address = myeth_addr;
         assert poolinfo[0].lptoken_address = lpt_call_addr;
-        assert poolinfo[0].staked_capital = Math64x61.fromFelt(5);
-        assert poolinfo[0].unlocked_capital = Math64x61.fromFelt(5);
+        assert poolinfo[0].staked_capital = Uint256(low=5000000000000000000, high=0);
+        assert_uint256_eq(poolinfo[0].unlocked_capital, Uint256(5000000000000000000, 0));
```

Comment:
@tensojka `assert poolinfo[0].staked_capital = Uint256(low=5000000000000000000, high=0);` should also be done as `assert_uint256_eq(poolinfo[0].staked_capital, Uint256(5000000000000000000, 0));` I guess :)